client/components/credit-card/stored-card.scss

```diff
@@ -0,0 +1,46 @@
+// Credit Card Payment Box
+// -----------------------------------
+
+.credit-card__stored-card {
+	background-image: url('/calypso/images/upgrades/cc-visa-disabled.svg');
+	background-position: 18px center;
+	background-repeat: no-repeat;
+	background-size: 60px auto;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	padding: 15px 15px 15px 95px;
+
+	.credit-card__stored-card-name {
+		color: $gray-darken-10;
+	}
+
+	.credit-card__stored-card-number {
+		color: $gray-dark;
+		flex-basis: 100%;
+		font-weight: 600;
+	}
+
+	.credit-card__stored-card-expiration-date {
+		color: $gray-darken-10;
+		font-style: italic;
+		opacity: 0.7;
+		white-space: nowrap;
+	}
+}
+
+$cc-brands: 'amex', 'diners', 'discover', 'jcb', 'mastercard', 'unionpay', 'visa';
```

Comment:
We have a finite number of these, I'd say it's be okay to list these out unless you feel very strongly about it strongly. This helps when performing searches in code. For example if I were to look up `.credit-card__stored-card.amex` I wouldn't get a result.

Reply:
I don't have super strong feelings, but I tend to avoid concat type code to help readers search later on. I try to optimize for lookup/reading over LoC savings within reason, since folks will be reading more than writing. In this case I'm fine with either approach, so don't count this as a blocker at all.